### PR TITLE
Revert change that added owner by default to filters that were passed to getAllStrategiesWithFilters from getUserPositions

### PR DESCRIPTION
**Pull request by markpradhan**

### CODE REVIEW

**Review by silviutroscot — APPROVED** [review]

File: packages/kamino-sdk/src/Kamino.ts [hunk 1]
```diff
@@ -5071,33 +5072,97 @@ export class Kamino {
...
+      const mintData = this._deserializeMint(mintInfo.data);
+
+      if (mintData.mintAuthority !== null && mintData.mintAuthority.equals(expectedMintAuthority)) {
+        kaminoMintWithAccountInfo.push([mint, accountInfo.account]);
       }
     }
+
+    const strategies = await batchFetch(
+      kaminoMintWithAccountInfo,
+      (chunk) => Promise.all(chunk.map(([mint]) => this.getStrategyByKTokenMint(mint))),
```
Comment (peroxy) [inline packages/kamino-sdk/src/Kamino.ts:5143]: This needs to make a get multiple accounts as well, dont Promise.all multiple requests. We have existing methods that fetch multiple strategies with batching already in the SDK

File: packages/kamino-sdk/src/Kamino.ts [hunk 2]
```diff
@@ -5071,33 +5072,97 @@ export class Kamino {
...
+    );
+
+    const positions: KaminoPosition[] = [];
+
+    for (const index of strategies.keys()) {
+      const strategy = strategies[index];
+      const accountData = kaminoMintWithAccountInfo[index][1];
+
+      if (!strategy || !accountData) {
+        console.error('Could not retrieve strategy');
```
Comment (peroxy) [inline packages/kamino-sdk/src/Kamino.ts:5154]: Remove logs from SDK

**Review by peroxy — CHANGES_REQUESTED** [review]

File: packages/kamino-sdk/src/Kamino.ts (hunk 1 above)
Comment (markpradhan) [reply packages/kamino-sdk/src/Kamino.ts]: updated

File: packages/kamino-sdk/src/Kamino.ts (hunk 2 above)
Comment (markpradhan) [reply packages/kamino-sdk/src/Kamino.ts]: updated

**Review by peroxy — APPROVED** [review]

**Pull request merged**